docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc

```diff
@@ -13,6 +13,10 @@ For previous rule updates, please navigate to the https://www.elastic.co/guide/e
 |Update version |Date | New rules | Updated rules | Notes
 
 
+|<<prebuilt-rule-8-12-20-prebuilt-rules-8-12-20-summary, 8.12.20>> | 21 Aug 2024 | 1 | 183 | 
+This release includes new rule for Linux. New rule for Linux include detection for defense evasion. Additionally, significant rule tuning for Windows, Linux and AWS integration rules has been added for better rule efficacy and performance.
```

Comment:
Please change first 2 sentences to "This release includes a new rule for Linux defense evasion detection."